### DISCUSSION TITLE
Selector @page not work

### DISCUSSION THREAD

**Discussion by JavierYucraZ** [mpdf/mpdf#1674] [General]
I need to modify the margins of the page with the selector @page, but it does not work for me in version 8

`@page {
        margin-left: 12mm;
        margin-right: 12mm;
        margin-top: 12mm;
        margin-bottom: 12mm;
    }`

This worked in version 6

**Comment by MAGGIx1404** [comment]
you can use  **@page** selector like this:

```

   /* default for all pages */
    @page{
        margin: 2in;   
    }
 
    /* margin on left page */
    @page :left {
        margin: 1in;   
    }
     
    /* margin on right page */
    @page :right {
        margin: 3in;   
    }
     
    /* top margin on first page */
    @page :first {
        margin-top: 5in;   
    }`
```

**Comment by finwe** [comment]
@JavierYucraZ `@page` CSS definition works normally in v 8. Show your setup for others to be able to see quirks in your setup.
